app/assets/javascripts/conference_form.js

```diff
@@ -0,0 +1,23 @@
+$(function () {
+  $('#new_conference').on('ajax:success',
+    function(e, data) {
+      var conference = JSON.parse(data)[0];
```

Comment:
See my comment about the return value from the controller in the `ConferencesController`. In my opinion, it's semantically confusing to return an array, plus I think
```js
JSON.parse(data)
```
also makes it more obvious that we're dealing with a single record.

Reply:
See my comment in the end